ui/src/apps/searchLogs/components/utils.ts

```diff
@@ -139,3 +140,17 @@ export function getAddress(target: LogsearchSearchTarget | undefined): string {
 }
 
 export const ServerTypeList = [ServerType.TiDB, ServerType.TiKV, ServerType.PD]
+
+export async function req_with_err_prompt<T>(
+  req: Promise<T>
+): Promise<T | null> {
+  try {
+    return await req
+  } catch (error) {
+    let msg = error?.response?.data?.message
+    if (msg) {
+      message.error(msg)
```

Comment:
Show error message directly may confuse user.
We'd better prompt some human-readable error message.
e.g.
`Failed to get download token for selected logs.`
`Failed to start a new search task, please try again later.`

Reply:
Agree, I'll change the function to accept a message parameter.